src/Options/Applicative/Types.hs

```diff
@@ -90,9 +91,9 @@ data ParserInfo a = ParserInfo
   { infoParser :: Parser a    -- ^ the option parser for the program
   , infoFullDesc :: Bool      -- ^ whether the help text should contain full
                               -- documentation
-  , infoProgDesc :: Chunk Doc -- ^ brief parser description
-  , infoHeader :: Chunk Doc   -- ^ header of the full parser description
-  , infoFooter :: Chunk Doc   -- ^ footer of the full parser description
+  , infoProgDesc :: Chunk AnsiDoc -- ^ brief parser description
+  , infoHeader :: Chunk AnsiDoc   -- ^ header of the full parser description
+  , infoFooter :: Chunk AnsiDoc   -- ^ footer of the full parser description
```

Comment:
This is still AnsiDoc, so that existing interface or these definitions doesn't change and so that users can continue using ansi styles while defining these.